### ISSUE TITLE
formatter.js regex

### ISSUE THREAD

**Issue by Mordef** [aswinkumar863/smarty-vscode-support#8]
as of #5 

some changes in client/src/formatter.js 
the regionTag:
```
const regionTag = {
  foldStartRegex: /{{?(block|capture|for|foreach|function|if|literal|section|setfilter|strip|while)[^}]*}?/,
  foldMiddleRegex: /{{?(else(if)?|foreachelse)[^}]*}?/,
  foldEndRegex: /{{?\/(block|capture|for|foreach|function|if|literal|section|setfilter|strip|while)[^}]*}?/,
  // foldBothRegex: /{{?\/?(block|capture|for|foreach|function|if|literal|section|setfilter|strip|while|else|elseif|foreachelse)[^}]*}?/g
  foldBothRegex: /{xxx}/g
};
```
I use ```[^}]*``` instead of ```\b*?}```and ```\b}```

'foldBothRegex' is for what?

the while loop i change to for
```
// while (lines[i]) {
for (i=0;i<lines.length;i++) {
// i++;
```

Now the examples i mentioned in #5 are working.

**Comment by aswinkumar863** [comment]
#5 is not reproduced. Help me in that here https://github.com/aswinkumar863/smarty-vscode-support/issues/5#issuecomment-641772792

**Comment by aswinkumar863** [comment]
The recommended fix will not be accepted because issue #5 is invalid.
